src/MediaWiki2DokuWiki/MediaWiki/Namespace/Page.php

```diff
@@ -1,4 +1,5 @@
 <?php
+require_once(DOKU_INC . "inc/common.php");
```

Comment:
Adding a `require_once` here introduces a side effect to the file. See PSR-1 section 2.3 on this.

Instead, can you move this to `MediaWiki2DokuWiki_DokuWiki_Bootstrap`? Add "common.php" to the `$files` array in the constructor after the "init.php" element.

```php
$files = array('init.php', 'common.php');
```